assets/extensions/extensions.scss

```diff
@@ -52,6 +52,7 @@ $wordpress-break: 961px;
 	&__grid {
 		display: flex;
 		flex-wrap: wrap;
+		align-items: flex-start;
```

Comment:
In a first version it didn't have the box heights aligned (see screenshot in https://github.com/Automattic/sensei/pull/4162). But the https://github.com/Automattic/sensei/pull/4163/ PR added some tweaks, including this. And I think the `align-items: flex-start` makes it not aligned again. Since we're planning to have dynamic pages, I think it's important to have it well aligned, so we avoid gaps between the blocks.